corehq/apps/export/const.py

```diff
@@ -21,3 +27,5 @@
 
 FORM_EXPORT = 'form'
 CASE_EXPORT = 'case'
+
+MAIN_TABLE = None
```

Comment:
yeah ~should be as simple as changing that. if it isn't, we're probably doing something wrong haha